packages/story-editor/src/components/panels/document/taxonmies/stories/index.js

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import styled from 'styled-components';
+/**
+ * Internal dependencies
+ */
+import TaxonomiesPanel from '../taxonomies';
+
+const Wrapper = styled.div`
+  background: ${({ theme }) => theme.colors.bg.primary};
+`;
+
+export default {
+  title: 'Stories Editor/Components/Taxonomies',
+  component: TaxonomiesPanel,
+};
+
+function Taxonomies() {
+  return <TaxonomiesPanel />;
+}
+
+export const _default = () => {
+  return (
+    <Wrapper>
+      <Taxonomies />
+    </Wrapper>
+  );
+};
```

Comment:
💭 Is the `Taxonomies` component needed? Lmk if this is a better pattern than letting the `_default` story render the `<TaxonomiesPanel />` without the wrapper

Reply:
Really I was just following patterns found in other storybook demos. My initial thought is / was that this demo would get flushed out once we get some designs on what this component will contain. Since I am not sure what all that will be I was thinking the separate `Taxonmies` component _could_ be useful. But thinking about it now, probably best to remove the extra component and the styled `Wrapper,` as well and pass in `rest` so we can style `TaxonomiesPanel` directly. ie. add background color.

Reply:
Whatever works 👍 I like the `<Wrapper/>` component for styling too. Just a small nit 😅